frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-external/details-card.tsx

```diff
@@ -105,13 +93,11 @@ export const DetailsCard: React.FC<DashboardItemProps> = ({
         {link && (
           <DetailItem
             key="ceph-dashboard-url"
-            title={t('ceph-storage-plugin~Ceph dashboard')}
+            title={t('ceph-storage-plugin~Ceph Dashboard')}
             isLoading={false}
             error={false}
```

Comment:
Sorry, I missed this earlier,
instead of directly passing `false`... we can use:
```
isLoading = !cephSecret?.loaded;
error = cephSecret?.loadError;
```